Metadata.yaml

```diff
@@ -0,0 +1,246 @@
+    that does not or cannot have any science papers and results associated with
+    it. Through authorship on the SpECTRE DOI such work gains recognition that
+    holds value in academia. Authorship on the SpECTRE DOI does not directly
+    entitle a person for authorship rights on other publications. Authorship
+    rights on publications related to SpECTRE are governed by our publication
+    policy.
+
+    Note that the placement of authors in these tiers is inherently a subjective
+    topic since we are forced to compare different projects people work on.
+    Nevertheless, having guidelines and some way of promoting healthy and
+    constructive discussions is vital for ensuring the collaborative aspect of
+    the code is healthy.
+
+  Core:
+    Description: |
+      For the time a Developer serves as Core developer they take on some
+      project-management responsibilities, such as merging pull-requests.
+
+      Guidelines on becoming a Core developer:
+
+      - 4-5 major infrastructure contributions, each taking roughly 4-5 weeks of
+        full-time work for an existing Core developer. Or, ∼20-25 weeks of
+        full-time infrastructure work saved for a current Core developer.
+      - Seeks out PRs to review and volunteers when calls for a reviewer are
+        made.
+      - Most people will not end up in the Core developer list since a lot of
+        work will lead to science results and authorship on science papers,
+        which pay dividends for the work.
+      - After a Core developer leaves the project they will remain in this tier
+        for at least as long as they were already in the Core tier before they
+        left, or for three years, whichever is longer. At that point the Core
+        developers will discuss if the person should be moved to the Developer
+        tier, where they will stay indefinitely. A Core developer can of course
+        also step down from the Core developer role at any time.
+
+      The author list order is decided by the Core developers.
+    List:
+      - Name: Deppe, Nils
+        Orcid: 0000-0003-4557-4115
+        Affiliations: [*caltech]
+        GitHub: nilsdeppe
+      - Name: Throwe, William
+        Orcid: 0000-0001-5059-4378
+        Affiliations: [*cornell]
+        GitHub: wthrowe
+      - Name: Kidder, Lawrence E.
+        Orcid: 0000-0001-5392-7342
+        Affiliations: [*cornell]
+        GitHub: kidder
+      - Name: Fischer, Nils L.
+        Orcid: 0000-0002-5767-3949
+        Affiliations: [*aei]
+        GitHub: nilsleiffischer
+
+  Developers:
+    Description: |
+      Developers have contributed major features to SpECTRE and remain in this
+      list indefinitely (unless they wish to be removed).
+
+      Guidelines on becoming a Developer:
+
+      - Do some code review, does not need to be self-initiated.
+      – Somewhere around 40 commits and 10k-15k lines of code should trigger a
```

Comment:
Inconsistent bullet dash-like thing.

Reply:
Ahh thanks for spotting this, I had weird formatting issues and didn't understand what caused that..